### ISSUE TITLE
a few small improvements in the Dutch translation

### ISSUE THREAD

**Issue by Rubensky1** [sabrogden/Ditto#559]
As the title says, I have made a few minor improvements to the Dutch translation of the tool.

Attached the improved Dutch.xml
[Ditto Dutch translation.zip](https://github.com/sabrogden/Ditto/files/13257131/Ditto.Dutch.translation.zip)

**Comment by Chealer** [comment]
Unless I am missing something, the only improvement is a full fix for the encoding issue reported in ticket #896.

**Comment by SonicWim** [comment]
Yes, correct. And one word, that is actually a more correct way of translation it

So as far as I'm concerned, this is solved with ticked [https://github.com/sabrogden/Ditto/issues/896](896)

Thanks!
